zetaclient/chains/bitcoin/client.go

```diff
@@ -6,7 +6,8 @@ import (
 	"github.com/zeta-chain/node/zetaclient/chains/bitcoin/signer"
 )
 
-type Client interface {
+//nolint:revive
+type BitcoinClient interface {
```

Comment:
Why these changes? Client is already be identified as bitcoin since in the Bitcoin package